AirportDictionary/AirportDictionary/Pages/Shared/_Layout.cshtml

```diff
@@ -17,7 +17,19 @@
     </environment>
     <link rel="stylesheet" href="~/css/site.css" />
 </head>
-<body>
+
+
+<body style="margin-top: 50px; background-color: white; color: black; font-weight: bold ">
+    <img src="~/css/photo.jpg" style="position: fixed;
```

Comment:
Nice styling, but move to a linked stylesheet so that it can be applied elsewhere.